rep-2006.rst

```diff
@@ -0,0 +1,73 @@
+We will work with you on a best-effort basis to help connect you with responsible parties best suited to address your concerns.
+
+How to submit a vulnerability
+=============================
+
+Please submit vulnerability reports by emailing security@openrobotics.org, preferably in English.
+This is a private list of security officers who can help verify the bug report and develop and release a fix.
+The more information provided about the bug, the easier it will be to fix.
+If you already have a fix, please include it with your report.
+We may ask to exchange PGP keys to discuss sensitive details about the vulnerability which may include proof-of-concept code, an impact assessment, recommended remediation steps or other information that will help us find and fix the problem.
+
+Include any plans you may have to disclose details about the vulnerability.
+In order to protect our users, unless otherwise agreed by both parties, we ask that you not publicly discuss the vulnerability until at least 90 days have passed or the fix is published.
+The group handling the report will also follow this timeline.
+If you do not wish to be acknowledged in the release communications please indicate so when you submit the vulnerability.
+
+What to expect in response to a vulnerability disclosure
+========================================================
+
+Expect a timely response to your notification, normally within two business days, during which time we will triage your vulnerability report.
+
+After we have evaluated the vulnerability we will send you our risk assessment and, where applicable, the anticipated time to publish an update.
+Where appropriate, we will coordinate and assign a vulnerability ID with an industry-standard.
+We will aim at being as transparent as possible about timelines and any issues or challenges which may impact the scheduled fix.
+
+Should the vulnerability have a widespread impact that requires coordinated disclosure, we may, at our discretion, choose to engage a neutral third party such as CERT/CC or ICS-CERT to support coordination efforts.
+
+We will notify you when a pull request has been submitted and approved which remediates the vulnerability.
+Because security vulnerabilities can have severe consequences, they differ from other bugs and at our discretion-- in order to protect our users-- we may modify our normal code release processes for publishing open source code updates.
+
+Safe Harbor
+===========
+
+Open Robotics strongly supports security research into ROS software and seeks to encourage that research.
+Open Robotics will not engage in legal action against individuals who act in good faith to identify, report and fix vulnerabilities in ROS, so long as they operate in accordance with any applicable laws or this policy.
+Unauthorized research or testing against operating robotic systems is in violation of this policy and strongly discouraged due to potential health and human safety concerns.
```

Comment:
The term of `Unauthorized` doesn't seem to fit. This makes it sound like prior clearance is required.
E.g. How would a third party gain proper authorization to research or test ROS?

Reply:
I took this to mean "Unauthorized by the owner/operator of the robot", but I'll add a little something to clarify this.

Reply:
52ddc96